.github/workflows/build.yml

```diff
@@ -0,0 +1,45 @@
+name: build
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        BUILD_TYPE: ["Debug"]
+        os: [ windows-latest ]
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Get MSVC
```

Comment:
This is not required since we write rust not c++

Reply:
I think this is for the requirement: "service fabric runtime installation"